cloud/pipeline.go

```diff
@@ -0,0 +1,81 @@
+package cloud
+
+import (
+	"context"
+	pb "github.com/earthly/cloud-api/pipelines"
+	"github.com/pkg/errors"
+)
+
+type PipelineTrigger struct {
+	Type     string
+	Modifier string
+}
+
+type PipelineArg struct {
+	Name         string
+	DefaultValue string
+}
+
+type Pipeline struct {
+	Repo          string
+	Path          string
+	Name          string
+	Org           string
+	Triggers      []*PipelineTrigger
+	Args          []*PipelineArg
+	RepoId        string
+	Project       string
+	IsPush        bool
+	Id            string
```

Comment:
nit: go linters usually suggests `ID`, I'm surprised our linter didn't complain on this. If the lint is passing :shrug: